Makefile

```diff
@@ -31,27 +31,43 @@ manager: generate fmt vet
 	go build -o bin/manager ./cmd/manager/main.go
 
 # Run against the configured Kubernetes cluster in ~/.kube/config
-run: install
-	$(KUSTOMIZE) build config/local_run | kubectl apply -n $(NAMESPACE) -f -
+run: install install_crd install_rbac
 	eval $$(scripts/dev/get_e2e_env_vars.py $(cleanup)); \
 	go run ./cmd/manager/main.go
 
 # Install CRDs into a cluster
-install: manifests kustomize
-	$(KUSTOMIZE) build config/crd | kubectl apply -f -
+install: manifests helm install_crd
+	
+
+install_crd:
+	kubectl apply -f config/crd/bases/mongodbcommunity.mongodb.com_mongodbcommunity.yaml
+
+install_rbac:
+	$(HELM) template -s templates/tests/operator_roles.yaml helm-chart | kubectl  -n $(NAMESPACE) apply -f -
+	$(HELM) template -s templates/tests/database_roles.yaml helm-chart | kubectl  -n $(NAMESPACE) apply -f -
+
+install_manager:
+	kubectl apply -f -n $(NAMESPACE) config/manager/manager.yaml
+
+uninstall_crd:
+	k delete crd mongodbcommunity.mongodbcommunity.mongodb.com
+
+uninstall_rbac: 
+	$(HELM) template -s templates/tests/operator_roles.yaml helm-chart | kubectl delete -f -
+	$(HELM) template -s templates/tests/database_roles.yaml helm-chart | kubectl delete -f -
 
 # Uninstall CRDs from a cluster
-uninstall: manifests kustomize
-	$(KUSTOMIZE) build config/crd | kubectl delete -f -
+uninstall: manifests helm uninstall_rbac uninstall_crd
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-deploy: manifests kustomize
-	cd config/manager && $(KUSTOMIZE) edit set image quay.io/mongodb/mongodb-kubernetes-operator=$(IMG):latest
-	$(KUSTOMIZE) build config/default | kubectl apply -n $(NAMESPACE) -f -
+deploy: manifests helm
+	install_crd
+	install_rbac
+	install_manager
```

Comment:
previously we were dynamically subbing in a value from `IMG` (which comes from the json config file). What it looks like this will do is just apply manager.yaml, is that correct?

One other thing to note, I see that you have using several helm template commands, one for each template file, we don't necessarily need to separate these out this much. We could do a single `helm install helm-chart --set key1=val1 --set key2=val2` etc.

Reply:
we also need to install the CRD and RBAC as well here I think. Because in the current version of the makefile we call `$(KUSTOMIZE) build config/default | kubectl apply -n $(NAMESPACE) -f -` which install the manager but also the crd and rbac